src/rules/imports.ts

```diff
@@ -83,6 +83,7 @@ const rules: Linter.RulesRecord = {
     },
   ],
   'import-x/prefer-default-export': 'error',
+  'import-x/prefer-namespace-import': 'error',
```

Comment:
It looks like without the `pattern` configuration it doesn't have sense. Let's put `{pattern: ["react", "react-dom"]}` in there or disable it with comment: "Disabled by default. Enable if needed"